--- ChangeLog	2024-10-14 08:42:43.068076737 -0400
+++ ChangeLog-spell-corrected	2024-10-14 08:42:14.890088208 -0400
@@ -67,7 +67,7 @@
 
 2024-10-04  rocky <rocky@gnu.org>
 
-	* setup-python-3.2.sh, setup-python-3.3.sh, setup-python-3.6.sh: 
+	* setup-python-3.2.sh, setup-python-3.3.sh, setup-python-3.6.sh:
 	Administrivia - go over setup-python
 
 2024-09-28  rocky <rocky@gnu.org>
@@ -76,7 +76,7 @@
 
 2024-09-28  rocky <rocky@gnu.org>
 
-	* admin-tools/merge-for-2.4.sh, admin-tools/setup-python-2.4.sh: 
+	* admin-tools/merge-for-2.4.sh, admin-tools/setup-python-2.4.sh:
 	Administrivia -- Add merge and setup for Python 2.4
 
 2024-09-28  rocky <rocky@gnu.org>
@@ -86,8 +86,8 @@
 
 2024-09-21  rocky <rocky@gnu.org>
 
-	* pyproject.toml: Remove setuptools version because...  older versions of Python don't support newer onces and setuptools
-	has vulnerabilites listed against it.  So newer Pythons use newer versions and older Pythons use what they
+	* pyproject.toml: Remove setuptools version because...  older versions of Python don't support newer ones and setuptools
+	has vulnerabilities listed against it.  So newer Pythons use newer versions and older Pythons use what they
 	can get.
 
 2024-09-15  rocky <rocky@gnu.org>
@@ -98,12 +98,12 @@
 
 	* admin-tools/{merge-for-3.2.sh => merge-for-3.3.sh},
 	admin-tools/{pyenv-3.2-3.5-versions => pyenv-3.3-3.5-versions},
-	admin-tools/{setup-python-3.2.sh => setup-python-3.3.sh}: 
+	admin-tools/{setup-python-3.2.sh => setup-python-3.3.sh}:
 	Administrivia
 
 2024-09-15  R. Bernstein <rocky@users.noreply.github.com>
 
-	* tracer/__init__.py, tracer/tracefilter.py, tracer/tracer.py: 
+	* tracer/__init__.py, tracer/tracefilter.py, tracer/tracer.py:
 	tracefilter corrections to ignore modules (#7) tracefilter corrections to ignore modules
 
 2024-09-14  rocky <rocky@gnu.org>
@@ -119,7 +119,7 @@
 2024-09-10  rocky <rocky@gnu.org>
 
 	* test/test_filter.py, test/test_tracer.py, tracer/tracefilter.py,
-	tracer/tracer.py: Rewite. "is_included" is now "is_excluded"
+	tracer/tracer.py: Rewrite. "is_included" is now "is_excluded"
 
 2024-09-10  R. Bernstein <rocky@users.noreply.github.com>
 
@@ -319,7 +319,7 @@
 
 2009-01-11  rocky.bernstein <rocky.bernstein@6ebf6a92-b7d9-11dd-8622-756dacf128ff>
 
-	* __pkginfo__.py, setup.cfg, test/test-basic.py, tracer/tracer.py: 
+	* __pkginfo__.py, setup.cfg, test/test-basic.py, tracer/tracer.py:
 	trace.py: Add EVENT2SHORT. Others: administrivia git-svn-id: https://pytracer.googlecode.com/svn/trunk@44
 	6ebf6a92-b7d9-11dd-8622-756dacf128ff
 
@@ -411,7 +411,7 @@
 
 2008-12-19  rocky.bernstein <rocky.bernstein@6ebf6a92-b7d9-11dd-8622-756dacf128ff>
 
-	* setup.py, test/test-basic.py, tracer/.gitignore, tracer/tracer.py: 
+	* setup.py, test/test-basic.py, tracer/.gitignore, tracer/tracer.py:
 	More interfaces made public. More work to back-patch caller frames.
 	Will probably simplify/generalize interface more as we use more.  git-svn-id: https://pytracer.googlecode.com/svn/trunk@27
 	6ebf6a92-b7d9-11dd-8622-756dacf128ff
@@ -544,7 +544,7 @@
 
 2008-11-21  rocky.bernstein <rocky.bernstein@6ebf6a92-b7d9-11dd-8622-756dacf128ff>
 
-	* .cvsignore, dbgcore.py, test/.cvsignore, test/basic.py, tracer.py: 
+	* .cvsignore, dbgcore.py, test/.cvsignore, test/basic.py, tracer.py:
 	Centralized Trace management around sys.settrace. We allow several
 	sets of trace events to get registered and unregistered and allow
 	tracing to be turned on and off temporarily without losing the trace
@@ -555,4 +555,3 @@
 
 	* Initial directory structure.  git-svn-id: https://pytracer.googlecode.com/svn/trunk@1
 	6ebf6a92-b7d9-11dd-8622-756dacf128ff
-
